website/content/docs/platform/k8s/vso/installation.mdx

```diff
@@ -24,26 +24,87 @@ HashiCorp helm repository and ensure you have access to the chart:
 
 ```shell-session
 $ helm repo add hashicorp https://helm.releases.hashicorp.com
+
 "hashicorp" has been added to your repositories
+```
 
+```shell-session
 $ helm search repo hashicorp/vault-secrets-operator
+
 NAME           	CHART VERSION	APP VERSION	DESCRIPTION
 hashicorp/vault-secrets-operator	0.3.1       	0.3.1     	Official HashiCorp Vault Secrets Operator Chart
 ```
 
 Then install the Operator:
 
 ```shell-session
-$ helm install --create-namespace --namespace vault-secrets-operator vault-secrets-operator hashicorp/vault-secrets-operator --version 0.3.1
+$ helm install --version 0.3.1 --create-namespace --namespace vault-secrets-operator vault-secrets-operator hashicorp/vault-secrets-operator
 ```
 
 
 ## Upgrading using helm
 
-Upgrades may be performed with `helm upgrade` on an existing install. Please
-always run Helm with `--dry-run` before any install or upgrade to verify
+You can upgrade an existing installation with the `helm upgrade` command.
+Please always run Helm with the `--dry-run` option before any install or upgrade to verify
 changes.
 
+Update the `hashicorp` Helm repo:
+```shell-session
+$ helm repo update hashicorp
+
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "hashicorp" chart repository
+Update Complete. ⎈Happy Helming!⎈
+```
+
+<note title="Helm does not automatically update CRDs">
+  You must update all CRDs manually before upgrading VSO.
+  Refer to <a href="#updating-the-crds">Updating CRDs</a>.
+</note>
```

Comment:
"Note" apparently needs to be capitalized to render properly, and the href needs updating.

```suggestion
<Note title="Helm does not automatically update CRDs">
  You must update all CRDs manually before upgrading VSO.
  Refer to <a href="#updating-crds">Updating CRDs</a>.
</Note>
```